torch/utils/cpp_extension.py

```diff
@@ -885,9 +964,13 @@ def win_wrap_ninja_compile(sources,
                 cuda_cflags=cuda_cflags,
                 cuda_post_cflags=cuda_post_cflags,
                 cuda_dlink_post_cflags=cuda_dlink_post_cflags,
+                sycl_cflags=None,
+                sycl_post_cflags=None,
+                sycl_dlink_post_cflags=None,
                 build_directory=output_dir,
                 verbose=True,
-                with_cuda=with_cuda)
+                with_cuda=with_cuda,
+                with_sycl=False)
```

Comment:
Why sycl doesn't support on windows + ninja?

Reply:
In this PR - by intent. I focused on implementing this support for Linux as the first step. In followed up PR this support might need to be extended for Windows.